projects/tasks/AdapterTasks.wdl

```diff
@@ -725,4 +729,33 @@ task CreateStagingAreaFile {
     File is_update_file = "staging_area.json"
   }
 
+}
+
+task ValidateStagingArea {
+  input {
+
+    String staging_area
+    Boolean done
+
+    String docker = "us.gcr.io/broad-gotc-prod/hca-adapter-tools:main_1637242455"
+    Int cpu = 1
+    Int memory_mb = 1000
+    Int disk_size_gb = 1
+  }
+
+  command{
+
+    validate-staging-area \
+    --staging-area ~{staging_area} \
+    --ignore-dangling-inputs 
+
+    }
+
+  runtime {
+    docker: docker
+    cpu: cpu
+    memory: "${memory_mb} MiB"
```

Comment:
Are we standardized on using the ~{ format?

Reply:
ah all the tasks in this WDL are using the ${, should I switch? I have no preference

Reply:
according to this I think we're supposed to use $ notation?

https://github.com/broadinstitute/warp/blob/develop/website/docs/contribution/contribute_to_warp/wdl_task_runtime_style.md